src/groups/mqb/mqbblp/mqbblp_storagemanager.cpp

```diff
@@ -1037,6 +1043,8 @@ void StorageManager::recoveredQueuesCbImpl(
         mqbs::ReplicatedStorage* rs = storageMapIt->second;
         BSLS_ASSERT_SAFE(rs);
 
+        const bool isStrongConsistency = !rs->hasReceipt(bmqt::MessageGUID());
```

Comment:
Looks a rather hacky way to determine whether a message is under strong consistency. But I am fine with it if you don't see any better way.

Reply:
If you stick with this way, perhaps add a `isStrongConsistency(const bmqt::MessageGUID& guid)` method somewhere to make the intentions clear.

Reply:
d